docker/docker-compose.ltc.yml

```diff
@@ -0,0 +1,54 @@
+version: '2'
+services:
+    # ltc is an image of litecoin node which used as base image for ltcd and
+    # ltcctl. The environment variables default values determined on stage of
+    # container start within starting script.
+    ltcd:
+      image: ltcd
+      container_name: ltcd
+      build:
+        context: ltcd/
+      volumes:
+        - shared:/rpc
+        - litecoin:/data
+      environment:
+        - RPCUSER
+        - RPCPASS
+        - NETWORK
+        - DEBUG
+        - MINING_ADDRESS
+      entrypoint: ["./start-ltcd.sh"]
+
+    lnd:
+      image: lnd
+      build:
+        context: ../
+        dockerfile: docker/lnd/Dockerfile
+      environment:
+        - RPCUSER
+        - RPCPASS
+        - NETWORK
+        - CHAIN
+        - DEBUG
+      volumes:
+        - shared:/rpc
+        - lnd:/root/.lnd
+      entrypoint: ["./start-lnd.sh"]
+      links:
+        - "ltcd:blockchain"
+
+volumes:
+  # shared volume is need to store the btcd rpc certificates and use it within
+  # btcctl and lnd containers.
+  shared:
+    driver: local
+
+  # litecoin volume is needed for maintaining blockchain persistence
+  # during ltcd container recreation.
+  litecoin:
+    driver: local
+
+  # lnd volume is used for persisting lnd application data and chain state
+  # during container lifecycle.
+  lnd:
```

Comment:
To avoid collision with the other lnd, name this volume `lnd_ltc` (change also in the `volumes` section above).